### PR TITLE
[DOC-273] Added formatting info and SQL Data Type supported

### PR DESCRIPTION
**Pull request by nadegepepin**

### CODE REVIEW

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 1]
```diff
@@ -192,3 +193,58 @@ You have run a query using SQL Ingest. How do you inspect its result?
...
+
+    Note that the trailing Z (UTC) is truncated from Timestamps without Ttmezones and from Dates. 
+
+    ***Example***
+
+    |Type|Database|CSV|JSON|
+    |----|--------|---|----|
+    |Date|2022-05-09|2022-05-09T00:00:00|"2022-05-09T00:00:00"|
+    |Timestamp ntz|2022-05-09 16:43:00|2022-05-09 16:43:00|"2022-05-09 16:43:00"|
+    |Timestamp tz|2022-05-09 16:43:00 -05:00|2022-05-09 16:43:00 -05:00|"2022-05-09 16:43:00 -05:00"|
```
Comment (albscui) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:234]:
```suggestion
    |Timestamp tz|2022-05-09 16:43:00 -05:00|2022-05-09T16:43:00-05:00|"2022-05-09T16:43:00-05:00"|
```

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 2]
```diff
@@ -192,3 +193,58 @@ You have run a query using SQL Ingest. How do you inspect its result?
...
+
+The following comments on formatting reflect the state of this release and are subject to change.
+
+### SQL datatypes supported
+
+We support the following SQL datatypes. Some of those Data Types are specific to databases.
+
+| Dates/Timestamps | Varchars | Numerics | Booleans |
+|------------------|---------|----------|----------|
+|`DATE` <br> `TIME`<br> `TIMESTAMP`<br> `TIMESTAMP_LTZ`<br> `TIMESTAMP_NTZ`<br> `TIMESTAMP_TZ`<br> `TIMESTAMPTZ`<br> `TIMESTAMP WITH TIME ZONE`<br> `TIMESTAMP WITHOUT TIME ZONE` |`VARCHAR`<br> `TEXT`<br> `CHARACTER VARYING`|`SMALLINT`<br> `INT2`<br> `INTEGER`<br> `INT`<br> `INT4`<br> `BIGINT`<br> `INT8`<br>`FLOAT`<br> `FLOAT4`<br> `FLOAT8`<br> `REAL`<br> `DOUBLE PRECISION`<br>`NUMERIC`<br> `DECIMAL`<br> `NUMBER`<br> `FIXED`|`BOOL`<br>`BOOLEAN`|
```
Comment (albscui) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:207]: `FIXED` is not a database type, it is a type specific to Snowflake returned by the Go driver.

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 3]
```diff
@@ -192,3 +193,58 @@ You have run a query using SQL Ingest. How do you inspect its result?
...
+
+        |Database|CSV|
+        |--------|---|
+        |"null"|null|
+        |\`""\`|""""""|
+        |nil||
+        |`"my string"`|"""my string"""|
+        |"this will be enclosed in quotes because it has a ,"|"this will be enclosed in quotes because it has a ,"|
+
+    * JSON: Our JSON encoding does not support infinite (Inf) and not a number (NaN) values.
```
Comment (albscui) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:250]: Just to clarify, users can still store Inf and NaN in JSON, except they will just be stored as strings.

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 4]
```diff
@@ -192,3 +193,58 @@ You have run a query using SQL Ingest. How do you inspect its result?
...
+
+        ***Example***
+
+        |Database|CSV|
+        |--------|---|
+        |"null"|null|
+        |\`""\`|""""""|
+        |nil||
+        |`"my string"`|"""my string"""|
+        |"this will be enclosed in quotes because it has a ,"|"this will be enclosed in quotes because it has a ,"|
```
Comment (albscui) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:248]: We should also document the empty string case, which gets saved as `""` in CSV.

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md [hunk 5]
```diff
@@ -192,3 +193,58 @@ You have run a query using SQL Ingest. How do you inspect its result?
...
+- **Strings**
+
+    * CSV: Keep in mind when parsing your CSVs in your user code that we escape `"` with `""` in CSV files.
+
+        ***Example***
+
+        |Database|CSV|
+        |--------|---|
+        |"null"|null|
+        |\`""\`|""""""|
```
Comment (albscui) [inline doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:245]: I don't think we should quote these strings in backticks.

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md (hunk 1 above)
Comment (nadegepepin) [reply doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:234]: good catch.

File: doc/docs/master/how-tos/basic-data-operations/sql-ingest.md (hunk 5 above)
Comment (nadegepepin) [reply doc/docs/master/how-tos/basic-data-operations/sql-ingest.md:245]: We do in markdown to display the `.

**Pull request merged**